src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -418,6 +419,7 @@ def process_inputs(self, inputs: TextGenerationInput) -> List[numpy.ndarray]:
         :param inputs: the input schema for the pipeline
         :return: the inputs for the engine
         """
+        self.streaming = inputs.streaming
```

Comment:
This is setting streaming for the whole pipeline, not just this single inference. I think we should move it to the context output from this function and pipe it everywhere to use it locally like the streamer object this is replacing

Reply:
good catch 100%

Reply:
reason for this was because need to know if streaming in the `join_engine_outputs` which doesn't have access to the context variable/would require further update to the engine code. Possible solution would be updating the outputs just that along with the finish reason, we're storing the streaming variable as well.

Reply:
required an update to the pipeline. updated